Asme migrated content for Category-Overview pages

ankur-dubey-rise opened the pull request:
Asme migrated content for Category-Overview pages :
BPVC-Resources, Educational-Partners, Policy-Publications, Engineering-History and Programs-Public-Policy-Society-Policies

luke80-riseinteractive commented on Category-Overview/BPVC-Resources.html:
```diff
@@ -1,47 +1,137 @@
+		list-style: none;
+	}
 
+	.primary-list li:before {
```
This is not needed. There are already blue bullets provided by the site style sheet.

luke80-riseinteractive commented on Category-Overview/BPVC-Resources.html:
```diff
@@ -1,47 +1,137 @@
+	</section>
 </div>
+<style>
+	h3,
```
You don't need to override the color of headings.

luke80-riseinteractive commented on Category-Overview/BPVC-Resources.html:
```diff
@@ -1,47 +1,137 @@
 
-<p><a class="arrow-blue" href="~/wwwasmeorg/media/ResourceFiles/AboutASME/Who%20We%20Are/BPVC%20Resources/Approval_of_New_Materials.pdf">Approval of New Materials</a> This is a guideline on how you may obtain approval of new material used for the construction of boilers, pressure vessels, &amp; nuclear components in accordance with the ASME Boiler &amp; Pressure Vessel Code.</p>
+<div class="gernral-page-template">
+	<section class="membership-content bpvc-resources">
```
This section tag is not required.

luke80-riseinteractive commented on Category-Overview/BPVC-Resources.html:
```diff
@@ -1,47 +1,137 @@
+</style>
 
-<p><a class="arrow-blue" href="~/wwwasmeorg/media/ResourceFiles/AboutASME/Who%20We%20Are/BPVC%20Resources/Approval_of_New_Materials.pdf">Approval of New Materials</a> This is a guideline on how you may obtain approval of new material used for the construction of boilers, pressure vessels, &amp; nuclear components in accordance with the ASME Boiler &amp; Pressure Vessel Code.</p>
+<div class="gernral-page-template">
```
This general content wrapper is not required.

luke80-riseinteractive commented on Category-Overview/BPVC-Resources.html:
```diff
@@ -1,47 +1,137 @@
-<div class="module full flyoutImage"><!-- lead image -->
-<div class="col-md-auto image-with-caption widget float-left figure" data-style-removed=" float-removed:right; margin-removed: -83px -44px 0px 0px;"><img alt="BPVC Resources" class="col-md-auto widget float-left figure" height="530" src="~/getmedia/30aba381-33a8-4a5a-9086-403549ede796/BPVC-Resources.aspx " width="389" /></div>
+<style>
```
Overriding the link color is a bad practice. Please remove.

luke80-riseinteractive commented on Content-Detail/Made-in-Japan-Earthquake-Proof-Homes.html:
```diff
@@ -1,31 +1,133 @@
+                                their business, for the duration of the quake. Then the airbag deflates and the house
+                                gently settles back down.</p>
+
+                            <aside class="content-aside mr-md-3 mr-0"
```
```suggestion
```
The `<figure>` should be enough, and this will work as intended when we have the `float-left` class added.

luke80-riseinteractive commented on Content-Detail/Made-in-Japan-Earthquake-Proof-Homes.html:
```diff
@@ -1,31 +1,133 @@
+                    <div class="col-md-12 col-12 mt-3">
+                        <h2>
+                            Minimizing Damage</h2>
+                        <aside class="content-aside mr-md-3 mr-0"
```
```suggestion
```
As above.

luke80-riseinteractive commented on Content-Detail/Made-in-Japan-Earthquake-Proof-Homes.html:
```diff
@@ -1,31 +1,133 @@
+                </div>
+                <div class="row">
+                    <div class="col-md-12 col-12 mt-3">
+                        <h2>
```
We should move the `<H2>` element below the floated figure. That way the text wraps cleanly around the content.

luke80-riseinteractive commented on Content-Detail/Made-in-Japan-Earthquake-Proof-Homes.html:
```diff
@@ -1,31 +1,133 @@
+                </div>
+                <div class="row">
+                    <div class="col-md-12 col-12 mt-3">
+                        <h2>
```
Consider same treatment as line 61

luke80-riseinteractive commented on Content-Detail/Making-3D-Skin-Printer-Portable.html:
```diff
@@ -1,33 +1,121 @@
+            <div class="content">
+                <div class="row">
+                    <div class="col-md-12 col-12 ">
+                        <article class="article-content1">
```
```suggestion
```
No need for additional containers...

ankur-dubey-rise commented:
We replaced all image's path with absolute value and removed article tags in all content details pages. let me know any of the page which does not have so we make sure right content is merged.

ankur-dubey-rise commented:
We got the issue there are 16 files of content-details which were not included in yesterday's push because of some confusion at our side ... I will push those 16 files as well today in 18-april branch.
